pkg/controller/usersignup/usersignup_controller.go

```diff
@@ -180,9 +180,18 @@ func (r *ReconcileUserSignup) Reconcile(request reconcile.Request) (reconcile.Re
 				return reconcile.Result{}, err
 			}
 		}
-
+		// look-up the `basic` NSTemplateTier to get the NS templates
+		var nstemplateTier toolchainv1alpha1.NSTemplateTier
+		err := r.client.Get(context.TODO(), types.NamespacedName{
+			Namespace: request.Namespace, // assume that NSTemplateTier were created in the same NS as Usersignups
+			Name:      "basic",
+		}, &nstemplateTier)
+		if err != nil {
+			reqLogger.Error(err, "No 'basic' NSTemplateTier found.")
```

Comment:
Let's update status here.

Reply:
done in https://github.com/codeready-toolchain/host-operator/pull/94/commits/bc9973a563e64570c9cbe8ed77fccdf2fa6e99ed

Reply:
See https://github.com/codeready-toolchain/host-operator/pull/94#discussion_r345906061